ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -596,20 +592,23 @@ const Completed = () => <FlexRow data-test-id='dar-completed' style={styles.comp
 
 interface CardProps {
   profile: Profile,
-  modules: Array<AccessModule>,
+  modules: AccessModule[],
```

Comment:
This is my personal preference, though there is no official style ruling on this in the codebase. However, I made the change here because this is the only instance of `Array<>` in the file, everything else is using `X[]`